### PR TITLE
Version info

### PR DESCRIPTION
**Pull request by streeve**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.16)
```
Comment (MattRolchigo) [inline CMakeLists.txt:1]: Was this change related?

File: src/Version.hpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#include <string>
+
+namespace Finch
+{
+
+//! Git version.
+inline std::string version() { return Finch_VERSION_STRING; }
+
+//! Git hash.
+inline std::string commitHash() { return Finch_GIT_COMMIT_HASH; }
```
Comment (MattRolchigo) [inline src/Version.hpp:41]: Do these functions actually get called anywhere?

**Review by MattRolchigo — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (streeve) [reply CMakeLists.txt]: No :(

File: src/Version.hpp (hunk 2 above)
Comment (streeve) [reply src/Version.hpp]: No :(

Comment (streeve) [reply src/Version.hpp]: Fixed now

**Review by MattRolchigo — APPROVED** [review]

**Pull request merged**